values-tls.yaml

```diff
@@ -0,0 +1,172 @@
+# Default values for infinispan-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+images:
+  # [USER] The container images for server pods.
+  server: quay.io/infinispan/server:14.0
+  initContainer: registry.access.redhat.com/ubi8-micro
+
+deploy:
+  # [USER] Specify the number of nodes in the cluster.
+  replicas: 1
+
+  clusterDomain: cluster.local
+
+  container:
+    extraJvmOpts: ""
+    libraries: ""
+    # [USER] Define custom environment variables using standard K8s format
+    # env:
+    #  - name: STANDARD_KEY
+    #    value: standard value
+    #  - name: CONFIG_MAP_KEY
+    #    valueFrom:
+    #      configMapKeyRef:
+    #        name: special-config
+    #        key: special.how
+    #  - name: SECRET_KEY
+    #    valueFrom:
+    #      secretKeyRef:
+    #        name: special-secret
+    #        key: special.how
+    env:
+    storage:
+      size: 1Gi
+      storageClassName: ""
+      # [USER] Set `ephemeral: true` to delete all persisted data when clusters shut down or restart.
+      ephemeral: false
+    resources:
+      # [USER] Specify the CPU limit and the memory limit for each pod.
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      # [USER] Specify the maximum CPU requests and the maximum memory requests for each pod.
+      requests:
+        cpu: 500m
+        memory: 512Mi
+
+  security:
+    secretName: ""
+    batch: ""
+  ssl:
+   secretName: "tls-secret"
```

Comment:
We should add this to `values.yaml` with an empty string.

Reply:
why we need this in values.yaml? iirc it's not used without encryption

Reply:
Including it in the `values.yaml` provides a reference of the fields available to users. We do this with many fields that don't have a value by default.

Reply:
oh ok